modules/quality_control/test/quality_controlTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * AcknowledgementsIntegrationTest automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Wang Shen <wangshen.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+ require_once __DIR__
+    . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+/**
+ * AcknowledgementsIntegrationTest
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Wang Shen <wangshen.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class QualityControlIntegrationTest extends LorisIntegrationTest
+{
+    /**
+     * Insert testing data into the database
+     *
+     * @return void
+     */
+    function setUp()
+    {
+        parent::setUp();
+    }
+    /**
+     * Delete test data
+     *
+     * @return void
+     */
+    function tearDown()
+    {
+        parent::tearDown();
+    }
```

Comment:
overriding the parent just to explicitly call the parent seems unnecessary

Reply:
Hahaha i just copied from the other PR.

Can someone push to my branch ?

Reply:
@ridz1208 I push the change.